src-tauri/src/event_handler/categorizer/mod.rs

```diff
@@ -5,31 +5,33 @@ use helpers::{inverted_index_categorizer, vscode_categorizer, youtube_categorize
 use crate::raw_metric_processor::{ActivityTag, ProcessedResult, StartActivity};
 
 #[derive(Debug, PartialEq)]
-pub struct Category(String);
+pub struct Category(pub String);
```

Comment:
why need to make the Category publish if you're going to `impl value` anyways :))

Reply:
Maybe should remove `pub`